docs/getting-started/basic-example.md

```diff
@@ -0,0 +1,181 @@
+#  Basic Collaborative Learning Setup
+
+Fed-BioMed enables users to quickly create a basic collaborative learning setup within seconds, providing an intuitive framework for exploring federated learning. This article serves as a comprehensive guide, walking you through the process of creating multiple nodes, designing a training plan for collaborative learning, and executing an experiment that performs model training.
+
+## Setup your working directory
+
+It is essential to set up a dedicated working directory in your local environment to organize all Fed-BioMed components. Additionally, create a `data` directory within the working directory to store the dataset that will be used in this guide.
+
+## Create your nodes
+
+The following two commands will create two node components. The directory that components will be initialized will be relative to the path that these commands are executed. Therefore, please make sure that you changed your directory to the working directory that you've created for Fed-BioMed.
+
+```
+fedbiomed component create -c node -p ./fbm-node
+fedbiomed component create -c node -p ./fbm-second-node
```

Comment:
Hi @ybouilla,

I don't think `-` is only for optional arguments. It can be for required arguments as well. Fed-BioMed CLI uses `argparser` module of Python. It uses `actions` and `arguments/options`. There, the action is `create` for a `component`, and the options are `-c/--component`: component type, `-p/--path` where will the component be instantiated. 

I understand that `fedbiomed component create node` makes sense. However, this is not introduced in this PR, command line options and actions was already like this one for a long time. I remember that I implemented that way in order to use single method in argument parser to manage component creation. It is because the component creation is quite similar for both component type.